composeApp/src/commonMain/kotlin/com/majotyler/hiittimer/presentation/common/navigation/NavigationRoot.kt

```diff
@@ -38,15 +45,17 @@ fun NavigationRoot() {
     NavDisplay(
         backStack = backStack,
         entryProvider = entryProvider {
+            entry<Route.AddWorkout> {
+                CreateWorkoutScreen()
+            }
```

Comment:
This is the Composable code that actually runs when we navigate to this new screen. There is no ViewModel yet so we don't need to create it here. When we add a ViewModel, we will need to create it here. :)